README.md

```diff
@@ -6,13 +6,13 @@ This project is based on the AddressBook-Level3 project created by the [SE-EDU i
 
 * This is the CS2103T Project for **Team CS2103T-W10-2**, the name of the project is StoreMando.
 
-* This product is a centralized inventory manager that helps to track the location of items in a household, their
-  individual quantities and respective expiry dates (if applicable). <br>
-  The usage of this product can eventually extend to other places outside households, however the core of our
-  development will surround the usage in a household.
-
-* It is named `StoreMando`, a combination of the words Storage and Commando, the goal of our product is to make it the
-  go-to product to track inventory.
+* StoreMando is an integrated platform fully customised for residents in households, residential colleges and halls,
+  with the aim of helping you manage your items effectively and efficiently. StoreMando allows you to keep track of your
+  items' whereabouts, quantities and expiry dates with a few simple commands. Furthermore, StoreMando also comes with an
+  inbuilt reminder feature to help you keep track of items that are expiring. All your items' information is
```

Comment:
Similarly, I think we should avoid using first person in this section and write it more formally.